SonarQube.Bootstrapper/Program.cs

```diff
@@ -18,6 +19,14 @@ private static int Main(string[] args)
         {
             var logger = new ConsoleLogger();
             BuildAgentUpdater updater = new BuildAgentUpdater();
+
+            if (false)
+            {
+                Console.WriteLine("Hello, world!");
+
+                // Console.WriteLine("Commented out code!");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=csharpsquid%3AS125)